examples/Demo/Demo.csproj

```diff
@@ -1,21 +1,21 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <OutputType>exe</OutputType>
-    <TargetFrameworks>net45</TargetFrameworks>
-    <IsPackable>false</IsPackable>
-    <DebugType>Full</DebugType>
-    <ProjectConfigFileName>App.config</ProjectConfigFileName>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="loggly-csharp" Version="4.6.1.82" />
-    <PackageReference Include="loggly-csharp-config" Version="4.6.1.82" />
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
-  </ItemGroup>
-
-  <ItemGroup>
-    <ProjectReference Include="..\..\src\NLog.Targets.Loggly\NLog.Targets.Loggly.csproj" />
-  </ItemGroup>
-
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>exe</OutputType>
+    <TargetFrameworks>net45</TargetFrameworks>
+    <IsPackable>false</IsPackable>
+    <DebugType>Full</DebugType>
+    <ProjectConfigFileName>App.config</ProjectConfigFileName>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="loggly-csharp" Version="4.6.1.88" />
+    <PackageReference Include="loggly-csharp-config" Version="4.6.1.88" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
Can you change the Newtonsoft.Json to 11.0.2 ?

Reply:
Sure thing. But first would you mind taking a look at this [PR](https://github.com/neutmute/loggly-csharp/issues/106)?

There's an issue with the build that needs to come in first.

Reply:
@neutmute Likes to publish to MyGet instead of NuGet, so you cannot upgrade the NLog-project. Also while testing, then there is no need to update NLog-project. You can just reference both projects. Then others will not be affected by your testing (By taking latest dependencies and suddenly having things break).

Reply:
@snakefoot got it...regardless though there is a build issue that should definitely be fixed

`C:\projects\loggly-csharp\source\loggly\LogglyClient.cs(174,30): error CS1061: 'Type' does not contain a definition for 'IsAssignableFrom' and no accessible extension method 'IsAssignableFrom' accepting a first argument of type 'Type' could be found (are you missing a using directive or an assembly reference?) [C:\projects\loggly-csharp\source\NetStandard\Loggly\Loggly.csproj]
334
335
    2 Warning(s)
336
    1 Error(s)
337
338
Time Elapsed 00:00:08.18
339
Execute Tests`

Please have a look at the linked PR and you will see. This build error has existed for a few weeks now.

Reply:
merged and published 
https://www.nuget.org/packages/loggly-csharp/4.6.1.90

Reply:
@neutmute should be all set now. Thanks!

Reply:
Thank you for your contribution @amikapoor !